manim/config/utils.py

```diff
@@ -32,6 +32,10 @@ def config_file_paths():
     return [library_wide, user_wide, folder_wide]
 
 
+def get_project_root() -> Path:
+    return Path(__file__).parent.parent.parent
```

Comment:
Hmm. This will return the folder where the `manim` directory is(usually in `site-packages` if installed by pip. Do you think 🤔 is this any helpful to a user?

Reply:
Same thought. :-)

Reply:
Not for users, but for devs.
E.g. in the test section that would be convenient to use, see e.g. here:
https://github.com/ManimCommunity/manim/blob/5de13530df7f8213580c0a772ee97552fb0dda66/tests/test_graphical_units/test_img_and_svg.py#L14

Reply:
Library code *is* for users. If it is not supposed to be used, by users, don't put it in the library.

Reply:
I would rather do a different thing in test rather than exporting this function(I know exporting is meaningless is python). You could simply use the thing @behackl specified.

Reply:
> Library code _is_ for users. If it is not supposed to be used, by users, don't put it in the library.

Ok, I see, I did not think about having a space that is not user-facing (e.g. the /tests folder).
That makes a lot of sense!
What do you think about moving it here:
https://github.com/ManimCommunity/manim/tree/master/scripts